docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -0,0 +1,1183 @@
+        SERVICE_FEE_ESTIMATOR_CACHE: 'redis://localhost:6379/1',
+        ENABLE_FEE_ESTIMATOR_QUICK: 'true',
+        ENABLE_FEE_ESTIMATOR_FULL: 'false',
+      },
+    },
+    {
+      name: 'lisk-service-transaction-statistics',
+      script: 'app.js',
+      cwd: './services/transaction-statistics',
+      pid_file: './pids/service_transaction_statistics.pid',
+      out_file: './logs/service_transaction_statistics.log',
+      error_file: './logs/service_transaction_statistics.err',
+      log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
+      watch: false,
+      kill_timeout: 10000,
+      max_memory_restart: '300M',
+      autorestart: true,
+      env: {
+        // --- Remember to set the properties below
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_STATISTICS_REDIS: 'redis://localhost:6379/1',
+        SERVICE_STATISTICS_MYSQL: 'mysql://lisk:password@localhost:3306/lisk',
+        TRANSACTION_STATS_HISTORY_LENGTH_DAYS: '366',
+      },
+    },
+    {
+      name: 'lisk-service-market',
+      script: 'app.js',
+      cwd: './services/market',
+      pid_file: './pids/service_market.pid',
+      out_file: './logs/service_market.log',
+      error_file: './logs/service_market.err',
+      log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
+      watch: false,
+      kill_timeout: 10000,
+      max_memory_restart: '300M',
+      autorestart: true,
+      env: {
+        // --- Remember to set the properties below
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_MARKET_REDIS: 'redis://localhost:6379/2',
+        SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB',
+        SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF',
+        // EXCHANGERATESAPI_IO_API_KEY: ''
+      },
+    },
+    {
+      name: 'lisk-service-export',
+      script: 'app.js',
+      cwd: './services/export',
+      pid_file: './pids/service_export.pid',
+      out_file: './logs/service_export.log',
+      error_file: './logs/service_export.err',
+      log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
+      watch: false,
+      kill_timeout: 10000,
+      max_memory_restart: '300M',
+      autorestart: true,
+      env: {
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_EXPORT_REDIS: 'redis://localhost:6379/3',
+        SERVICE_EXPORT_REDIS_VOLATILE: 'redis://localhost:6379/4',
+      },
+    },
+  ],
+};
+----
+====
+
+== Microservice Configurations
+In this section, we discuss the common and specific configurations of all the microservices of Lisk Service.
+
+=== Common Configurations
+The common configurations described in this section can be added to any microservice.
+The value of each configuration defined below should be the same across all microservices.
+
+[tabs]
+=====
+Properties::
++
+--
+[cols="2,1,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Configurable
+|Description
+|Sample
+
+| `SERVICE_BROKER`
```

Comment:
Below `SERVICE_BROKER`, you could also specify `SERVICE_BROKER_TIMEOUT`.

Reply:
Please share the following for the `SERVICE_BROKER_TIMEOUT`

- Datatype
- Description
- Sample

Reply:
Datatype: Number
Description: Inter-microservice request timeout in seconds
Sample: 10